mergify_engine/web_cli.py

```diff
@@ -81,7 +81,7 @@ def queues():
     args = parser.parse_args()
     asyncio.run(
         api_call(
-            config.BASE_URL + f"/queues/{args.owner_id}",
+            config.BASE_URL + f"/queues_v2/{args.owner_id}",
```

Comment:
if that is the only changes, I'm wondering if it's worth doing it at all

since you are then going to rename this `/queues_v2` as `/queues`, you'll get back to the same point…

Reply:
ok